conf/deployment/vsphere/upi_1az_rhcos_multus_vsan_3m_3w.yaml

```diff
@@ -13,6 +13,8 @@ ENV_DATA:
   compute_memory: '65536'
   fio_storageutilization_min_mbps: 10.0
   is_multus_enabled: true
+  multus_create_public_net: true
+  multus_create_cluster_net: true
```

Comment:
is the plan to create configuration files for covering more permutations of public net and cluster net?

Reply:
Yes, should have those configs in today.